features/XouBaseApiLoginAndRegister/Service/XouBaseApiCalls.swift

```diff
@@ -0,0 +1,49 @@
+//
+//  XouBaseApiCalls.swift
+//  XouDevSpec
+//
+//  Created by Xander Schoeman on 2020/04/19.
+//
+
+import Foundation
+
+
+public struct XouBaseApiCalls {
+
+    public init() {
+    }
+    
+    public  func getUsers(completetionHandler: @escaping(Result<[User], AnimeError>) -> Void) {
+        
+        
+            guard let url = URL(string: "https://xoubase.herokuapp.com/users") else {
```

Comment:
Consider keeping your URLs in a constant in a struct so that you could re-use it if needed and all URLs exist in one place

Reply:
Ok, ill change this as well.